### DISCUSSION TITLE
Log the same metric when using multiple dataloaders in validation_step()

### DISCUSSION THREAD

**Discussion by thangld201** [Lightning-AI/pytorch-lightning#18542] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
Hi, I have a particular use case when using multiple dataloaders for validation in `validation_step()`. I have metrics that I want to aggregate across these different dataloaders for checkpoint monitoring, but when I use `self.log('val/loss',....)` in the `validation_step()`, automatically a suffix will be added: e.g. `'val/loss'` -> `'val/loss/dataloader_idx_0'`, `'val/loss'` -> `'val/loss/dataloader_idx_1'`... This makes it impossible to reduce this metric across datasets. 

I tried setting `add_dataloader_idx=False` but it reports another error 

```
pytorch_lightning.utilities.exceptions.MisconfigurationException: You called 
`self.log(val/loss, ...)` twice in `validation_step` with different arguments. This is not allowed
```

How can I do this in Pytorch Lightning ? I'm currently using the 1.9.4 version.

**Comment by SeppeLampe** [comment]
It's been a while, but did you find a solution?
